src/main/java/emissary/pool/AgentThreadGroup.java

```diff
@@ -20,13 +20,6 @@ public void uncaughtException(Thread thread, Throwable throwable) {
             logger.error("Fatal ThreadDeath on " + thread.getName() + " had ThreadDeath exception", throwable);
             throw (java.lang.ThreadDeath) throwable;
         }
-        logger.error("#######    #    #######    #    #      ");
-        logger.error("#         # #      #      # #   #      ");
-        logger.error("#        #   #     #     #   #  #      ");
-        logger.error("#####   #     #    #    #     # #      ");
-        logger.error("#       #######    #    ####### #      ");
-        logger.error("#       #     #    #    #     # #      ");
-        logger.error("#       #     #    #    #     # #######");
         logger.error("Fatal Thread Error on " + thread.getName() + " had an uncaught exception", throwable);
```

Comment:
If we want this message to have greater cosmetic visibility, this could still use a multi-line message and ("i.e., "####################\nFatal Thread Error on ..."), but otherwise this looks good.

Reply:
That would certainly avoid the multiple logger calls which is the main issue, but the secondary objectives are to make it human readable and make it consumable downstream w/ log analysis tools. Changing the level to FATAL could achieve this as well, but I'm ok with the simplicity here and leaving it ERROR.

This should be convertible to parameterized logging. That would be my only ask. Bonus points for fixing both log statements.

Reply:
Added parameterized logging for both log statements in the most recent change